Realm/Tests/SwiftUITestHostUITests/SwiftUITestHostUITests.swift

```diff
@@ -60,6 +60,19 @@ class SwiftUITests: XCTestCase {
         }
     }
 
+    private func checkRowCount(_ table: XCUIElement, _ expected: Int, line: UInt = #line) {
```

Comment:
Would be possible to use this function to check all the row counts in all UI tests. Checking if the count is equal to the expected count in cases the cells fit in the screen and if not check the table frame height with the  lastCel origin + height like you do here
```
let lastCell = table.cells.element(boundBy: count - 1)
XCTAssertGreaterThan(lastCell.frame.origin.y + lastCell.frame.height, table.frame.height)
```